README.md

```diff
@@ -77,31 +77,30 @@ If you have installed an SSD and a HDD in your T430, you can gain the advantages
 		killAll Finder
 
 
-#### Install Clover
+#### Install OpenCore
 
-	- Install “clover.pkg” (found in repo/tools folder)
-    	Check 'Install for UEFI Booting Only'
-		This will automatically check 'Install Clover in the ESP'
-		Check Drivers 64 UEFI (IMPORTANT! Do only check OsxAptioFix2Drv-64)
+	-> Obtain latest build of [OpenCorePkg](https://github.com/acidanthera/OpenCorePkg)
+		* (Preferred) Build from latest source with `macbuild.tool`
+		* Download [latest pre-compiled release](https://github.com/acidanthera/OpenCorePkg/releases/latest)
+		
+	-> Extract the EFI folder from OpenCore to the root of the EFI partition
 
-	- Now you can remove your Installer Pen Drive (you have boot)
 
-
-#### Configure EFI/Clover
+#### Configure EFI/OC
 	- config.plist: use the one from this repository
-        take from low-resolution-config.plist folder for 1366x768 display
-		take from high-resolution-config.plist folder for 1600x900 or + display
+		- High-resolution -> `high-resolution-config.plist/config.plist`
+		- Low-resolution -> `low-resolution-config.plist/config.plist`			
 
-	- /drivers64UEFI
-		use the folder from this repository (with 4 files only!)
+	- /Drivers
+		* Use the folder from this repository (with 5 files only!)
 
-	- /kexts
-		Remove everything except other (leave it empty)
+	- /Kexts
+		* Add kexts from this repository 
 
-	- /ACPI/patched:
+	- /ACPI:
 		(OPTION 1 Difficult - but recommended) Make your own SSDT and DSDT:
 			-> First you'll need the vanilla .aml generated by your UEFI (BIOS)
-				When in the clover boot menu press F4 (and Fn+F4 just in case)
+				When in the *clover* boot menu press F4 (and Fn+F4 just in case)
```

Comment:
How are users reaching this menu, if they do not install Clover?

Reply:
Fair enough. OpenCore doesn't have this feature, though. Maybe there will need to be a step for booting from a Clover flash drive? Not sure if there's another way to get DSDT/SSDT besides through clover.

Reply:
You can grab a copy from UEFI shell or using SSDTime which is recommended from creating the various SSDTs you'll need. You can read more about it in the [Dortania ACPI docs](https://dortania.github.io/Getting-Started-With-ACPI/Manual/dump.html#from-opencore).